source/blog/001-how-extend-works.md

```diff
@@ -0,0 +1,206 @@
+---
+title: How @extend Works
+author: Natalie Weizenbaum
+tags: blog
+layout: base
```

Comment:
We can move these shared pieces of data (plus `permalink`) into a `source/blog/blog.11tydata.yml` file.